perf/stability/stability.mk

```diff
@@ -0,0 +1,25 @@
+WD = ./perf/stability
+STABILITY = $(WD)/setup_test.sh
+
+# Standard set of stability tests to run
+stable_tests = http10 graceful-shutdown gateway-bouncer mysql redis
```

Comment:
`nit: stability_tests`

Reply:
This isn't a typo, this is the stable stability tests. The others are flakey or require not standard setup. Very confusing though